pyaedt/modules/DesignXPloration.py

```diff
@@ -456,29 +487,38 @@ def add_dx_setup(self,variables_to_include, defaults_var_values=None, setupname=
 
 
 class ParametericsSetups(object):
-    """Parametric Setup Class"""
+    """ParametricsSetups class."""
     class Setup(CommonOptimetrics, object):
```

Comment:
```suggestion
    class Setup(CommonOptimetrics, object):

```

Reply:
FYI @PipKat, this is a class inside a class.  Depending on how this is exposed to the user, this might be internal enough that we don't have to document the class itself since they can't create it.